packages/api-client-core/src/GadgetConnection.ts

```diff
@@ -69,7 +72,7 @@ export class GadgetConnection {
   private websocketsEndpoint: string;
   private websocketImplementation: any;
   private _fetchImplementation: typeof fetch;
-  private environment: "Development" | "Production";
```

Comment:
this is used for `getCurrentSpan()?.setAttributes({ applicationId: this.options.applicationId, environmentName: this.environment });`. not sure if sure if there are concerns that the case here changes